js/infobar.js

```diff
@@ -0,0 +1,466 @@
+'use strict';
+
+var $infoBar = {};
+var $message = {};
+var $acceptButton = {};
+var $cancelButton = {};
+var bars = {};
+var opened = false;
+
+var bar = {};
+var json = {};
+var lang = '';
+var availableLangs = {};
+var acceptLangs = [];
```

Comment:
One suggestion is to upcase all these global vars. Once you get in to the code, it becomes difficult to tell the difference between function scoped and global variables.

Also, would be good to wrap all JS in an IIFE to keep the global namespace clean.